packages/aws-client/src/SNS.ts

```diff
@@ -4,10 +4,96 @@
 
 import pRetry from 'p-retry';
 import Logger from '@cumulus/logger';
+import {
+  PublishCommand,
+  SubscribeCommand,
+  UnsubscribeCommand,
+  ListSubscriptionsByTopicCommand,
+  CreateTopicCommand,
+  DeleteTopicCommand,
+  ConfirmSubscriptionCommand,
+  PublishCommandInput,
+  SubscribeCommandInput,
+  UnsubscribeCommandInput,
+  ListSubscriptionsByTopicCommandInput,
+  CreateTopicCommandInput,
+  DeleteTopicCommandInput,
+  ConfirmSubscriptionCommandInput,
+} from '@aws-sdk/client-sns';
 import { sns } from './services';
 
 const log = new Logger({ sender: 'aws-client/sns' });
 
```

Comment:
These functions catch the original error and throw new Error object.  I think it's better to keep the original error, so these functions are not necessary.